api/lanes.py

```diff
@@ -854,8 +854,11 @@ class RelatedBooksLane(WorkBasedLane):
     * RecommendationLane: Works provided by a third-party recommendation
       service.
     """
+    CACHED_FEED_TYPE = CachedFeed.RELATED_TYPE
     DISPLAY_NAME = "Related Books"
     ROUTE = 'related_books'
+    MAX_CACHE_AGE = 48*60*60 # 48 hours, the shortest time of any component
+                             # sublane.
```

Comment:
Why is that? It doesn't seem like these would change very much.

Reply:
That's just what it was before. I bumped up the component cache times to 96 hours and turned this into an expression rather than a hard-coded value. (This required that I move RelatedBooksLane to lower in lanes.py, after the classes for its sublanes.)

Reply:
Working on my bugfix branch I realized it's more complicated than this. The "recommendations" part of this feed is set up to show books that are currently available. That information is going to go stale a lot faster than the rest of the feed.

If we had a way of caching the _recommendation data_ from NoveList, we could cache _that_ for a really long time, and cache the _feeds_ (which take availability into account) for a relatively short time.